messaging/amqp-reactive/src/test/java/io/quarkus/ts/messaging/amqpreactive/BaseAmqpReactiveIT.java

```diff
@@ -31,4 +31,5 @@ public void testLastPrice() {
                             "Expected prices not found in " + response);
                 });
     }
+
```

Comment:
Can you remove this line as well as I mention in comment https://github.com/quarkus-qe/quarkus-test-suite/pull/1749#discussion_r1579436800